py3status/modules/speedtest.py

```diff
@@ -25,11 +25,11 @@
     unit_size: unit for bytes_received/bytes_sent (default 'MB')
 
 Format placeholders:
-    {bytes_sent}            bytes sent during test, human readable, eg 'TODO'
+    {bytes_sent}            bytes sent during test, human readable, eg '52.45'
     {bytes_sent_unit}       unit used for bytes_sent, eg 'MB'
-    {bytes_sent_raw}        bytes sent during test, for format comparation, eg 'TODO'
-    {bytes_received}        bytes received during test, human readable, eg 'TODO'
-    {bytes_received_raw}    bytes received during test, for format comparation, eg 'TODO'
+    {bytes_sent_raw}        bytes sent during test, for format comparation, eg '52445184.0'
+    {bytes_received}        bytes received during test, human readable, eg '70.23'
```

Comment:
70.23 what?  70.23kb/s 70.23Tb/s

Reply:
You know I do sometimes a the code ;p

My point is that the example is not valid. Maybe this is not clear from my comment

Reply:
But the example is valid... is what I was trying to say.

If a CPU placeholder prints a percent number, then the example shouldn't show `%` too. And if it does, then the `eg` example is wrong.

If it actually print `%` too.. then the code is wrong. Sorry if I was not clear too.

Reply:
OK makes sense the documentation could be clearer

Reply:
Must I keep default placeholders `bytes_send` and `bytes_received` and create `*_human` one for `self.py3.safe_format_units`?